### PR TITLE
Added brick blocks, brick items in progress.

### PR DESCRIPTION
**Pull request by ImDaRealAni**
Added brick blocks.

### CODE REVIEW

**Comment by Sandertv** [comment]
Please feel free to add brick items in this PR too.

File: dragonfly/block/bricks.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+type Bricks struct {
+	noNBT
+	solid
+}
+
+// BreakInfo ...
+func (b Bricks) BreakInfo() BreakInfo {
+	return BreakInfo{
+		Hardness: 2,
+		Harvestable: func(t tool.Tool) bool {
```
Comment (DaPigGuy) [inline dragonfly/block/bricks.go:18]: pickaxeHarvestable is the equivalent to this function

File: dragonfly/item/register.go [hunk 2]
```diff
@@ -83,4 +83,5 @@ func init() {
 	world.RegisterItem("minecraft:clay_ball", ClayBall{})
 
 	world.RegisterItem("minecraft:stick", Stick{})
-}
+	world.RegisterItem("minecraft:bricks", Brick{})
```
Comment (Sandertv) [inline dragonfly/item/register.go:86]: I think this is supposed to be `minecraft:brick`

Comment (ImDaRealAni) [reply dragonfly/item/register.go:86]: Hm, it worked for me when I tested in the creative inventory.

Comment (Sandertv) [reply dragonfly/item/register.go:86]: Because this is used on disk and not over network.

File: dragonfly/block/bricks.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	solid
+}
+
+// BreakInfo ...
+func (b Bricks) BreakInfo() BreakInfo {
+	return BreakInfo{
+		Hardness: 2,
+		Harvestable: pickaxeHarvestable,
+		Effective: pickaxeEffective,
+		Drops: simpleDrops(item.NewStack(b, 1)),
```
Comment (Sandertv) [inline dragonfly/block/bricks.go:19]: Make sure to run go fmt over these files.

**Pull request closed without merging**